### PR TITLE
QG in red fixes

### PR DESCRIPTION
**Pull request by patogallaiovlabs**
Fixing sonar issues:
https://sonarcloud.io/dashboard?id=rskj

### CODE REVIEW

File: rskj-core/src/main/java/org/ethereum/util/ByteUtil.java [hunk 1]
```diff
@@ -366,14 +366,14 @@ public static String oneByteToHexString(byte value) {
     public static int numBytes(String val) {
 
         BigInteger bInt = new BigInteger(val);
-        int bytes = 0;
+        int bytes = 1;
```
Comment (Vovchyk) [inline rskj-core/src/main/java/org/ethereum/util/ByteUtil.java:369]: how about this way?
```
    public static int numBytes(String val) {
        return new BigInteger(val).bitLength() / 8 + 1;
    }
```

File: rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java [hunk 2]
```diff
@@ -77,9 +78,9 @@ public void stop() {
     }
 
     private void closeAll() {
-        logger.trace("closing blockStore.");
+        getLogger().trace("closing blockStore.");
```
Comment (nicops) [inline rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java:81]: I don't think this is the right way to tackle this. Instead, I would change the logging level of the test logback.xml

File: rskj-core/src/main/java/org/ethereum/util/ByteUtil.java [hunk 3]
```diff
@@ -366,14 +366,14 @@ public static String oneByteToHexString(byte value) {
...
 
         while (!bInt.equals(BigInteger.ZERO)) {
             bInt = bInt.shiftRight(8);
             ++bytes;
         }
-        if (bytes == 0) {
```
Comment (nicops) [inline rskj-core/src/main/java/org/ethereum/util/ByteUtil.java:375]: SonarQube says this expression is always true, but that seems off. Surely it is not guaranteed it is always true, given the while above? bad static analysis?

File: rskj-core/src/main/java/org/ethereum/util/ByteUtil.java (hunk 1 above)
Comment (patogallaiovlabs) [reply rskj-core/src/main/java/org/ethereum/util/ByteUtil.java:369]: worked like a charm!

File: rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java (hunk 2 above)
Comment (patogallaiovlabs) [reply rskj-core/src/main/java/co/rsk/core/bc/BlockChainFlusher.java]: changed as suggested, thanks!

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Review by Vovchyk — APPROVED** [review]

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Review by ajlopezrsk — APPROVED** [review]

**Pull request merged**